plugins/xivo-cisco-sccp/legacy/var/tftpboot/RINGLIST.XML

```diff
@@ -1,2 +1 @@
-<CiscoIPPhoneRingList>
-</CiscoIPPhoneRingList>
+RINGLIST.XML
```

Comment:
I just noticed this: could you do a separate PR for this and explain why? It's not in the same plugin. Thank you!

Reply:
It's an issue, you can remove it....